docs/EVM.md

```diff
@@ -0,0 +1,19 @@
+# Public Endpoints
+
+|Network |Type  | Endpoint| GasLimit
+--- | --- | ---| ---
+|kerria-test-net|JSON RPC|https://kerria-rpc.parallel.fi:29933| 15_000_000
```

Comment:
shall we use standard eth rpc port?

Reply:
guess this public endpoint can be configured by DevOps without exposing port,  and for local test sometime conflict cause trouble